lib/tapioca/compilers/dsl/active_record_associations.rb

```diff
@@ -153,6 +153,10 @@ def populate_associations(mod, constant)
             add_error(<<~MSG)
               Cannot generate association `#{reflection.name}` on `#{constant}` since the source of the through association is missing.
             MSG
+          rescue NameError
+            add_error(<<~MSG)
+              Cannot generate association `#{reflection.name}` on `#{constant}` since the constant `#{reflection.name.capitalize}` does not exist.
```

Comment:
Oh, we should also change this to use `class_name` as well:
```suggestion
              Cannot generate association `#{reflection.name}` on `#{constant}` since the constant `#{reflection.class_name}` does not exist.
```